forge-gui/res/cardsfolder/c/coral_helm.txt

```diff
@@ -4,4 +4,5 @@ Types:Artifact
 A:AB$ Pump | Cost$ 3 Discard<1/Random> | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +2 | NumDef$ +2 | SpellDescription$ Target creature gets +2/+2 until end of turn.
 SVar:NonStackingEffect:True
 AI:RemoveDeck:All
```

Comment:
maybe this could be replaced with a Discard Pref?

Reply:
Hmm the effect is random